### PR TITLE
dev-python/tqdm: new ebuild

### PR DESCRIPTION
**Pull request by TheChymera**
tqdm is a small python package adding progress status to iterators. It is used among others by moviepy, a video editing toolkit for python, for which an ebuild is currently in the making.

Maintainership request submitted: https://bugs.gentoo.org/show_bug.cgi?id=592382

### CODE REVIEW

File: dev-python/tqdm/ChangeLog [hunk 1]
```diff
@@ -0,0 +1,18 @@
+  29 Aug 2016; Horea Christian <h.chr@mail.ru> metadata.xml:
```
Comment (SoapGentoo) [inline dev-python/tqdm/ChangeLog]: Remove `ChangeLog`, we don't use those anymore in the pure git tree

File: dev-python/tqdm/tqdm-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/noamraph/tqdm"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
```
Comment (SoapGentoo) [inline dev-python/tqdm/tqdm-9999.ebuild]: make `DEPEND` fit on one line

File: dev-python/tqdm/tqdm-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_5)
```
Comment (SoapGentoo) [inline dev-python/tqdm/tqdm-9999.ebuild]: add space after python3_5, also consider py3.4 support

File: dev-python/tqdm/tqdm-0_pre20140115.ebuild [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+HOMEPAGE="https://github.com/noamraph/tqdm"
+COMMIT="da5fdbf7c05a9127f04ac5887292457e8335f480"
+SRC_URI="https://github.com/noamraph/${PN}/tarball/${COMMIT} -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE=""
+
+DEPEND="
```
Comment (SoapGentoo) [inline dev-python/tqdm/tqdm-0_pre20140115.ebuild]: same edits here

File: dev-python/tqdm/metadata.xml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>horea.christ@gmail.com</email>
+    <name>Horea Christian</name>
+  </maintainer>
```
Comment (SoapGentoo) [inline dev-python/tqdm/metadata.xml]: reformat like you did for the other PR

**Comment by idella** [comment]
vars IUSE and RDEPEND are empty and can go, be deleted. It is an optional change

**Comment by TheChymera** [comment]
@idella if it's an optional change, I'd prefer to have them explicitly empty.

**Comment by diogocp** [comment]
Development of tqdm now happens at https://github.com/tqdm/tqdm. This version is obsolete.

**Comment by SoapGentoo** [comment]
@diogocp send in a PR then?

**Pull request closed without merging**